coreneuron/nrniv/main1.cpp

```diff
@@ -387,42 +409,43 @@ extern "C" int run_solve_core(int argc, char** argv) {
 
     report_mem_usage("After mk_mech");
 
-    if (nrnopt_get_str("--report-conf").size()) {
-        if (nrnopt_get_int("--multiple") > 1) {
+    if (corenrn_param.reportfilepath.size()) {
+        if (corenrn_param.multiple > 1) {
             if (nrnmpi_myid == 0)
                 printf("\n WARNING! : Can't enable reports with model duplications feature! \n");
         } else {
-            configs = create_report_configurations(nrnopt_get_str("--report-conf").c_str(),
-                                                   nrnopt_get_str("--outpath").c_str());
+            configs = create_report_configurations(corenrn_param.reportfilepath.c_str(),
+                                                   corenrn_param.outpath.c_str());
             reports_needs_finalize = configs.size();
         }
     }
 
-    // initializationa and loading functions moved to separate
+    // initialization and loading functions moved to separate
     Instrumentor::phase_begin("load-model");
-    nrn_init_and_load_data(argc, argv, !configs.empty());
+    nrn_init_and_load_data(argc, argv, configs.size() > 0);
```

Comment:
why did you change this?

Reply:
I think it was a suggestion from @pramodk at some point during development. Is there anything wrong about it? Do you think it's an improvement?

Reply:
generally we prefer to use `empty()` over `size() == 0` since the former is more literal and shorter.

Reply:
I'll switch back to the old form then!